Gemfile

```diff
@@ -53,7 +54,6 @@ group :development, :test do
   gem 'rubocop-rails', require: false # A RuboCop extension focused on enforcing Rails best practices and coding conventions.
   gem 'rubocop-rspec', require: false # Code style checking for RSpec files
   gem 'rubocop-performance', require: false # An extension of RuboCop focused on code performance checks.
-  gem 'ffaker' # used to easily generate fake data: names, addresses, phone numbers, etc.
```

Comment:
Why we are removing it? 🤔  it easy to generate test data with it

Reply:
Because duplicated... It is still here [earlier in the GemFile](https://github.com/malparty/google-search-ruby/blob/f4fd89d668c0894e99a8a5323aaedfd21f4cd020/Gemfile#L11)